sql_generators/usage_reporting/templates/composite_active_users_aggregates.view.sql.jinja

```diff
@@ -37,11 +37,7 @@ FROM
   `{{ project_id }}.{{ app_name }}.active_users_aggregates`
   {% endif %}
 WHERE
-  {% if app_name == "fenix" %}
```

Comment:
I don't think this fenix specific behavior is needed - see [this chart](https://sql.telemetry.mozilla.org/queries/109302/source#268525) - we have usage reporting pings for fenix on versions 136, just like the other apps.